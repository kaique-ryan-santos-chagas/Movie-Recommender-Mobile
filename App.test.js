import React from 'react';

import renderer from 'react-test-renderer';

import MovieList from './src/pages/movieList';

jest.mock('react-native-gesture-handler');
jest.mock('react-native/Libraries/Animated/src/NativeAnimatedHelper');
jest.useFakeTimers();

test('generate', () => {
    const tree = renderer.create(<MovieList />).toJSON();
    expect(tree).toMatchSnapshot();
});

