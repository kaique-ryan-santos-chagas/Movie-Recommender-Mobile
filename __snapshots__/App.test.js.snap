// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generate 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#1C1C1C",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <ActivityIndicator
    animating={true}
    color="red"
    hidesWhenStopped={true}
    size={70}
  />
</View>
`;
